chartfx-dataset/src/test/java/de/gsi/dataset/spi/DataSetEqualityTests.java

```diff
@@ -219,41 +227,47 @@ public boolean canDelete(int index) {
         }
 
         @Override
-        public boolean isXEditable() {
-            return false;
-        }
-
-        @Override
-        public boolean isYEditable() {
+        public boolean isEditable(final int dimIndex) {
             return false;
         }
-
     }
 
-    private class OneDimDataSet extends AbstractDataSet implements DataSet {
+    private class OneDimDataSet extends AbstractDataSet<OneDimDataSet> implements DataSet {
+        private static final long serialVersionUID = 1L;
+        final double[] data = new double[] { 2.4, 5.2, 8.5, 9.2 };
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/GSI/chart-fx/pullRequest?prid=4477772)